### PR TITLE
fix: github issue 75, CLI error for variables outside prompt

### PR DESCRIPTION
**Pull request by brianpark0126**
only tested unofficially by using CLI commands (ADD and USE) within cookbook examples since Brendan has said I should wait for the official golden suite of CLI tests.

### CODE REVIEW

File: tests/cli/test_utils.py [hunk 1]
```diff
@@ -197,11 +197,19 @@ def test_get_prompt_analyzer():
...
     foo = "bar"
+    number = 1
+    settings = Settings()
+    a_list = [1, 2, 3]
     '''
     )
     analyzer = get_prompt_analyzer(sample_file_content)
     assert analyzer.comments == "This is a comment"
-    assert analyzer.variables == {"foo": "bar"}
+    assert analyzer.variables == {
```
Comment (brenkao) [inline tests/cli/test_utils.py:207]: just to make sure, when running `template.render` do we get back
```
number = 1
settings = Settings()
a_list = [1, 2, 3]
 ```

File: mirascope/cli/utils.py [hunk 2]
```diff
@@ -292,20 +292,22 @@ def _update_tag_decorator_with_version(
...
+def _update_mirascope_imports(imports: list[tuple[str, Optional[str]]]):
     """Updates the mirascope import."""
-    if not any(import_name == "mirascope" for import_name in imports):
-        imports.append("mirascope")
+    if not any(import_name == "mirascope" for import_name, _ in imports):
+        imports.append(("mirascope", None))
 
 
-def _update_mirascope_from_imports(member: str, from_imports: list[tuple[str, str]]):
+def _update_mirascope_from_imports(
```
Comment (brenkao) [inline mirascope/cli/utils.py:301]: There are two ` # TODO: Update `mirascope.tags` work with import alias` that need to be updated as well.

This use case needs to be supported: 
```
import mirascope as mira

@mira.tags()
class MyPrompt(Prompt):
    ...
```

Comment (brenkao) [reply mirascope/cli/utils.py:301]: Right now the strings that we are checking are hardcoded to be `mirascope.tags`

File: mirascope/cli/generic/prompt_template.j2
Comment (brenkao) [inline mirascope/cli/generic/prompt_template.j2:1]: update prompt_cli examples and other examples that use prompt_template

File: mirascope/cli/generic/mirascope.ini.j2
Comment (brenkao) [inline mirascope/cli/generic/mirascope.ini.j2:1]: update prompt_cli examples and other examples that use mirascope.ini

File: tests/cli/test_utils.py (hunk 1 above)
Comment (brianpark0126) [reply tests/cli/test_utils.py:207]: yes, the items in the dictionary thats passed in are all wrapped in an extra layer of string so you have to put that in this specific test, but when you render it with `template.render` it gets peeled away hence everything being a string and actual strings being double quoted

File: mirascope/cli/utils.py [hunk 3]
```diff
@@ -333,13 +340,20 @@ def write_prompt_to_template(
         }
 
     import_tag_name: Optional[str] = None
+    mirascope_alias = "mirascope"
+    for name, alias in analyzer.imports:
```
Comment (brenkao) [inline mirascope/cli/utils.py:344]: also need to check from_imports as well. e.g.
```
  from mirascope import tags as foo
```

**Comment by brenkao** [comment]
Fixes #75

**Pull request merged**